assets/airlines/apf.json

```diff
@@ -0,0 +1,19 @@
+{
+    "icao": "apf",  
+    "name": "Amapola Flyg",
+    "callsign": {
+        "name": "Amapola",
+        "callsignFormats": [
+            "####"
+        ]
+    },
+    "fleets": {
+        "default": [
+            ["F50", 8],
+            ["JS42", 1]
+        ],
+        "cargo": [
+            ["F50", 8]
```

Comment:
I'm not sure if I should use the same F50 for the "Fokker 50 Freighter" model which is used for cargo flights.